tests/resources/mir_datasets/compmusic_jingju_acappella/readme.txt

```diff
@@ -0,0 +1,31 @@
+Jingju a cappella singing dataset
```

Comment:
Why is the readme here?

Reply:
True. Not needed here. I have even removed it from `DOWNLOAD_REMOTES` since all the info about the dataset will be present in the docs.